javascript/javascript2/week2/movies.js

```diff
@@ -0,0 +1,162 @@
+const movies = [
+  { title: "'71", year: 2014, rating: 7.2, votes: 41702, running_times: 5940 },
+  {
+    title: "'A' gai wak",
+    year: 1983,
+    rating: 7.4,
+    votes: 11942,
+    running_times: 6300,
+  },
+  {
+    title: "'Breaker' Morant",
+    year: 1980,
+    rating: 7.9,
+    votes: 10702,
+    running_times: 6420,
+  },
+  {
+    title: "'Crocodile' Dundee II",
+    year: 1988,
+    rating: 5.5,
+    votes: 47180,
+    running_times: 6480,
+  },
+  {
+    title: "(500) Days of Summer",
+    year: 2009,
+    rating: 7.7,
+    votes: 412368,
+    running_times: 5700,
+  },
+  {
+    title: "*batteries not included",
+    year: 1987,
+    rating: 6.6,
+    votes: 25636,
+    running_times: 6360,
+  },
+  {
+    title: "...E tu vivrai nel terrore! L'aldilà",
+    year: 1981,
+    rating: 6.9,
+    votes: 16484,
+    running_times: 5220,
+  },
+  {
+    title: "...and justice for all.",
+    year: 1979,
+    rating: 7.4,
+    votes: 25408,
+    running_times: 7140,
+  },
+  { title: "10", year: 1979, rating: 6, votes: 13152, running_times: 7320 },
+  {
+    title: "10 Cloverfield Lane",
+    year: 2016,
+    rating: 7.2,
+    votes: 216151,
+    running_times: 6240,
+  },
+  {
+    title: "10 Items or Less",
+    year: 2006,
+    rating: 6.7,
+    votes: 13342,
+    running_times: 4920,
+  },
+  {
+    title: "10 Things I Hate About You",
+    year: 1999,
+    rating: 7.2,
+    votes: 247070,
+    running_times: 5820,
+  },
+  {
+    title: "10 Years",
+    year: 2011,
+    rating: 6.1,
+    votes: 20790,
+    running_times: 6000,
+  },
+  {
+    title: "10,000 BC",
+    year: 2008,
+    rating: 5.1,
+    votes: 114750,
+    running_times: 6540,
+  },
+  {
+    title: "100 Feet",
+    year: 2008,
+    rating: 5.5,
+    votes: 10979,
+    running_times: 6240,
+  },
+  {
+    title: "100 Girls",
+    year: 2000,
+    rating: 5.9,
+    votes: 19232,
+    running_times: 5640,
+  },
+  {
+    title: "101 Dalmatians",
+    year: 1996,
+    rating: 5.7,
+    votes: 83245,
+    running_times: 6180,
+  },
+];
+// Create an array of movies containing the movies with a short title
+const shortMovieTitle = movies.filter((movies) => movies.title.length < 10);
```

Comment:
Should be `movie` not `movies` since you already have an array named `movies`
```suggestion
const shortMovieTitle = movies.filter((movie) => movie.title.length < 10);
```